omod/src/main/webapp/resources/htmlforms/HMIS-ACP-CervicalCancerScreeningForm.xml

```diff
@@ -0,0 +1,155 @@
+
+            jq(document).ready(function () {
+                
+
+
+            });
+        }
+
+    </script>
+
+
+    <div>
+        <div class="row">
+            <div class="col-md-6">
+                <h1>HMIS ACP: Cervical Cancer Screening Form</h1>
+            </div>
+            <div class="col-md-5"></div>
+            <div class="col-md-1">
+                <submit class="confirm" submitLabel="Save" submitStyle="customSubmit"/>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-12">
+                <div class="card">
+                    <div class="card-header">
+                        Client Information
+                    </div>
+                    <div id="enrollment" class="card-body">
+                        <div class="row">
+                            <div class="col-4" id="visit-date">
+                                <label>Date</label>
+                                <encounterDate default="today"
+                                               disallowMultipleEncountersOnDate="block"/>
+                            </div>
+                            <div class="col-4">
+                                <label>Serial Number</label>
+                                <obs conceptId="1646" id="1646"/>
+                            </div>
+                            <div class="col-4">
+                                <label>Reg. No.</label>
+                                <obs id="168696" conceptId="168696"/>
+                            </div>
+                            
+                        </div>
+                        <div class="row">
+                            <div class="col-4">
+                                <label>Referred In</label>
+                                <obs id="168757" conceptId="168757" answerConceptIds="1065,1066" answerLabels="Yes, No" style="radio" class="horizontal"/>
+                            </div>
+                            <div class="col-4">
+                                <label>Entry point Code</label>
+                                <obs conceptId="164993"
+                                     answerConceptIds="160542,165047,165048,165046,165348,164984,90015,168695,  90018,99456,90002" 
+                                     answerLabels="OPD, ART Clinic, TB Clinic,Postnatal Clinic, Mother Baby Care Point, Family Planning Clinic,STI Clinic, Cancer Unit, In patient ward, Community/Outreach, Other Specify"
+                                     id="164993"/>
+                            </div>
+                            <div class="col-2">
+                                <label>Other Testing Point</label>
+                                <obs id="165398" conceptId="165398"/>
+                            </div>
+                            <div class="col-2">
+                                <label>Entry Point Clinic No.</label> 
+                                <obs id="168761" conceptId="168761"/>
```

Comment:
@semujju can't find this concept

Reply:
@semujju add a condition for these CaCx forms to only show on female clients only

Reply:
@semujju some fields are missing validations

Reply:
@Mwanje these requests have been effected